server/src/rooms/BattleRoom.ts

```diff
@@ -9,6 +9,12 @@ import {
 
 export class BattleRoom extends Room<MyRoomState> {
   maxClients = 4;
+  roundDurationMinutes = 1;
```

Comment:
Change the roundDuration Minutes to a constant style if it is not going to be modified, or maybe can instantiate it on creation by including it in the create room options when creating battleroom so that we can customise the round time.